src/py/flwr/cli/new/templates/app/pyproject.flowertune.toml.tpl

```diff
@@ -61,6 +61,9 @@ dataset.name = "$dataset_name"
 default = "local-simulation"
 
 [tool.flwr.federations.local-simulation]
+options.num-supernodes = 10
```

Comment:
```suggestion
```

Reply:
this is set dynamically based on the challenge selected for a flowertune app. (see next line in `pyproject.toml`).